CMakeLists.txt

```diff
@@ -5,8 +5,8 @@ SET (boolinq_VERSION_MAJOR 2)
 SET (boolinq_VERSION_MINOR 0)
 
 
-SET (CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS} -O0 -ggdb3 -DDEBUG -std=c++11")
-SET (CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS} -O3 -std=c++11")
+SET (CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS} -O0 -ggdb3 -DDEBUG")
+SET (CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS} -O3")
```

Comment:
I deduce that your compiler does not have `-std=gnu++11` option, or you don't like it. After reconsideration, I leave it to your decision. It is not difficult to make the little correction locally at all.